ush/set_extrn_mdl_params.sh

```diff
@@ -125,6 +125,9 @@ else
     "JET")
       EXTRN_MDL_SYSBASEDIR_ICS="${EXTRN_MDL_SOURCE_BASEDIR_ICS:-/mnt/lfs4/BMC/public/data/grids/gens/pgrb2b}"
       ;;
+    "HEAR")
```

Comment:
Please fix this typo - HERA?

Reply:
changed

Reply:
I still see "HEAR" in line 128. Is it changed?